crates/redstone/src/network/error.rs

```diff
@@ -108,6 +108,11 @@ pub enum Error {
     /// Includes FeedId that is reocuring.
     ConfigReocuringFeedId(FeedId),
 
+    /// Indicates that payload timestamps are not equal.
+    ///
+    /// Contains the timestamp that is not equal to the first one in the payload.
+    TimestampDifferentThanOthers(TimestampMillis),
```

Comment:
I would add here the other timestamp also, so we have more information in case of error